addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -3,7 +3,7 @@
 import json
 
 from odoo import _, api, fields, models
-
+from odoo.osv.expression import AND
```

Comment:
```suggestion
from odoo.osv.expression import AND

```
we're gonna need two empty lines before the `class` definition to pass the ci/style check